trace-dispatcher/src/Cardano/Logging/DocuGenerator.hs

```diff
@@ -354,21 +353,21 @@ documentMarkdown (Documented documented) tracers = do
     namespaceMetricsBuilder :: Namespace -> Builder
     namespaceMetricsBuilder ns = mconcat (intersperse (singleton '.') (map fromText ns))
 
-    propertiesBuilder :: LogDoc -> Builder
-    propertiesBuilder LogDoc {..} =
-        case nub ldSeverity of
-          []  -> fromText "> Severity:   " <> asCode (fromString (show Info))
-          [s] -> fromText "> Severity:   " <> asCode (fromString (show s))
-          l   -> fromText "> Severities: "
-                  <> mconcat (intersperse (singleton ',')
-                        (map (asCode . fromString . show) l))
-      <>
-        case nub ldPrivacy of
-          []  -> fromText "\nPrivacy:   " <> asCode (fromString (show Public))
-          [p] -> fromText "\nPrivacy:   " <> asCode (fromString (show p))
-          l   -> fromText "\nPrivacies: "
-                  <> mconcat (intersperse (singleton ',')
-                        (map (asCode . fromString . show) l))
+    -- propertiesBuilder :: LogDoc -> Builder
+    -- propertiesBuilder LogDoc {..} =
+    --     case nub ldSeverity of
+    --       []  -> fromText "> Severity:   " <> asCode (fromString (show Info))
+    --       [s] -> fromText "> Severity:   " <> asCode (fromString (show s))
+    --       l   -> fromText "> Severities: "
+    --               <> mconcat (intersperse (singleton ',')
+    --                     (map (asCode . fromString . show) l))
+    --   <>
+    --     case nub ldPrivacy of
+    --       []  -> fromText "\nPrivacy:   " <> asCode (fromString (show Public))
+    --       [p] -> fromText "\nPrivacy:   " <> asCode (fromString (show p))
+    --       l   -> fromText "\nPrivacies: "
+    --               <> mconcat (intersperse (singleton ',')
+    --                     (map (asCode . fromString . show) l))
```

Comment:
@jutaro, do we need this commented section?

Reply:
Removed, hope I can recover it one day